amicleaner/fetch.py

```diff
@@ -77,11 +77,17 @@ def fetch_unattached_lt(self):
 
         amis = []
         for lt_name in unused_lts:
-            resp = self.ec2.describe_launch_template_versions(
-                LaunchTemplateName=lt_name
-            )
-            amis.append(lt_latest_version.get("LaunchTemplateData", {}).get("ImageId")
-                        for lt_latest_version in resp.get("LaunchTemplateVersions", []))
+            try:
+                resp = self.ec2.describe_launch_template_versions(
+                    LaunchTemplateName=lt_name
+                )
+                amis.append(lt_latest_version.get("LaunchTemplateData", {}).get("ImageId")
+                            for lt_latest_version in resp.get("LaunchTemplateVersions", []))
+            except ClientError as e:
+                if e.response['Error']['Code'] == 'InvalidLaunchTemplateName.NotFoundException':
+                    print("Launch template just deleted {}. Skipping.".format(lt_name))
```

Comment:
Personally I'd prefer to say it the reason is that it is probably because it was just deleted.

Reply:
Yeh I changed it:
`print("Launch template just deleted {}. Skipping.".format(lt_name))`

No not necessarily just the silvers, the errors are occurring for the golds from the few broken builds so far.